core/src/map.cpp

```diff
@@ -216,30 +224,28 @@ void Map::loadSceneAsync(const char* _scenePath, bool _useScenePosition,
         nextScene = impl->nextScene;
     }
 
-    runAsyncTask([nextScene, _onMapReady, _onMapReadyUserData, _sceneUpdates, _onSceneUpdateError, this](){
+    runAsyncTask([nextScene, _onSceneReady, _sceneUpdates, this](){
 
-            bool newSceneLoaded = SceneLoader::loadScene(platform, nextScene, _sceneUpdates, _onSceneUpdateError);
+            bool newSceneLoaded = SceneLoader::loadScene(platform, nextScene, _sceneUpdates);
 
-            impl->jobQueue.add([nextScene, newSceneLoaded, _onMapReady, _onMapReadyUserData, _onSceneUpdateError, this]() {
+            impl->jobQueue.add([nextScene, newSceneLoaded, _onSceneReady, this]() {
```

Comment:
Userdata is required in cases like the tizen plugin where we need to pass the maps view handle (`maps_view_h`) to be used in the sceneReady callback.